rotkehlchen/db/schema.py

```diff
@@ -791,7 +791,7 @@
        rate,
        NULL AS fee, /* no fee */
        NULL AS fee_currency, /* no fee */
-       txhash AS link,
+       "0x" || lower(hex(txhash)) AS link,
```

Comment:
How slow does this make it? Have you measured before and after this PR with an account with many trades (both DEX and not )

Reply:
I've tested with randomized swaps and trades in the database (total of 6503 = 2500 trades + 4003 amm_swaps) and the old query takes around `0.01955` seconds on average and the new one `0.0214`. Tested both in a cold start.